backend/main/models/proposal.go

```diff
@@ -56,7 +57,12 @@ var computedStatusSQL = `
 	END as computed_status
 	`
 
-func GetProposalsForCommunity(db *s.Database, start, count int, communityId int, status string, order string) ([]*Proposal, int, error) {
+func GetProposalsForCommunity(
+	db *s.Database,
+	params shared.OrderedPageParams,
```

Comment:
Could we have the params consistently be the last parameter as in other function calls?